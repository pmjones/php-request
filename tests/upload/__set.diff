002- string(31) "SapiUpload::$name is read-only."
003- string(16) "RuntimeException"