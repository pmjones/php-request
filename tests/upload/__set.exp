string(16) "RuntimeException"
string(31) "SapiUpload::$name is read-only."
string(16) "RuntimeException"
string(43) "SapiUpload::$noSuchProperty does not exist."