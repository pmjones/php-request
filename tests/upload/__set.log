
---- EXPECTED OUTPUT
string(16) "RuntimeException"
string(31) "SapiUpload::$name is read-only."
string(16) "RuntimeException"
string(43) "SapiUpload::$noSuchProperty does not exist."
---- ACTUAL OUTPUT
string(16) "RuntimeException"
string(43) "SapiUpload::$noSuchProperty does not exist."
---- FAILED
