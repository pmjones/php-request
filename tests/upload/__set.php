<?php
$upload = new SapiUpload('foo', 'text/plain', 123, '/tmp/foo.txt', 0);
try {
    $upload->name = 'bar';
} catch( Exception $e ) {
    var_dump(get_class($e), $e->getMessage());
}
try {
    $upload->noSuchProperty = 'foo';
} catch( Exception $e ) {
    var_dump(get_class($e), $e->getMessage());
}
