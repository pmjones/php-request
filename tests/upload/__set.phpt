--TEST--
SapiUpload::__set
--FILE--
<?php
$upload = new SapiUpload('foo', 'text/plain', 123, '/tmp/foo.txt', 0);
try {
    $upload->name = 'bar';
} catch( Exception $e ) {
    var_dump(get_class($e), $e->getMessage());
}
try {
    $upload->noSuchProperty = 'foo';
} catch( Exception $e ) {
    var_dump(get_class($e), $e->getMessage());
}
--EXPECT--
string(16) "RuntimeException"
string(31) "SapiUpload::$name is read-only."
string(16) "RuntimeException"
string(43) "SapiUpload::$noSuchProperty does not exist."
