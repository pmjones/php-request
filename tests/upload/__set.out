string(16) "RuntimeException"
string(43) "SapiUpload::$noSuchProperty does not exist."